src/parser/jobs/gnb/modules/Cooldowns.tsx

```diff
@@ -0,0 +1,43 @@
+import ACTIONS from 'data/ACTIONS'
+import CoreCooldowns from 'parser/core/modules/Cooldowns'
+
+export default class Cooldowns extends CoreCooldowns {
+	static cooldownOrder = [
+		// Stance
+		ACTIONS.ROYAL_GUARD.id,
```

Comment:
hm... not sure how well this'll show up. given how tank stances work now, i should probably have a chat with you tank peeps about how best to display stances (if it's worth it at all? idk).